content/en/continuous_integration/setup_tests/agent.md

```diff
@@ -14,12 +14,9 @@ further_reading:
 <div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-To report test results to Datadog, the [Datadog Agent][1] is required.
+To report test result to Datadog, you can do it through the [Datadog Agent][1] (recommended for on-premises installations).
 
-There are two ways to set up the Agent in a CI environment:
-
-* Install the Agent as a long running process on each worker node (recommended for on-premises installations).
-* Run an ephemeral instance of the Agent as a service container on each build (recommended for SaaS CI providers).
+For SaaS CI providers, Datadog recommends using the [Agentless mode][7] of the Test Instrumentation Libraries. This mode allows sending data without the need of running an ephemeral instance of the Agent as a service container on each build.
```

Comment:
```suggestion
For SaaS CI providers, Datadog recommends using the [Agentless mode][7] of the Datadog Libraries. This mode allows sending data without running an ephemeral instance of the Agent as a service container on each build.
```
I'm not convinced by "test instrumentation libraries". Also some language simplification would be useful IMO